tests/test_babel.py

```diff
@@ -21,31 +21,10 @@ def test_init():
     """Test initialization."""
     d = MultidirDomain(paths=[".", ".."], entry_point_group="invenio_i18n.translations")
     assert d
-    assert d.has_paths()
+    assert len(d._translation_directories) > 0
```

Comment:
I guess normally we assume the MultidirDomain is tested in flask_babel, that's why removing the tests below is OK.

Reply:
MultidirDomain is not tested in flask_babel, because it is a class implemented in invenio-i18n. I removed the has_paths method from MultidirDomain, because it is only used within the tests. I see no point of having a method with the only purpose of testing it. Further the workflows has changed a little bit, before MultidirDomain was used in the LazyString which is not more the case. It is not more possible to do it that way, now the paths calculated by MultidirDomain are given to flask_babel.Babel in the ext.py